src/content/introduction.md

```diff
@@ -40,8 +40,12 @@
 
 ---
 
+![Redux Flow](content/images/full-redux-flow.png "Redux Flow")
+
+---
+
 ## Not in this talk
-- Redux 101
+- Intro to Redux
```

Comment:
I like this idea, I think the diagram is pretty dope and would help us a lot explaining briefly the concepts of redux.